### ISSUE TITLE
java.nio.file.InvalidPathException: Illegal char <:> when using karate runner with Junit5

### ISSUE THREAD

**Issue by hahampis** [kirksl/karate-runner#83]
I have setup the sample project as described [here](https://github.com/intuit/karate#quickstart). I can run `users.feature` from VSCode using both Runners, `UsersRunner.java` and `ExamplesTest.java`. However, trying to run it with Karate runner extension I get:

```
java.nio.file.InvalidPathException: Illegal char <:> at index 16: examples/users/c:\Users\hahampis\Github\myproject\src\test\java\examples\users\users.feature
```

The executed command as reported by VSCode is 

```
cmd.exe /d /c mvn clean test -f "c:\Users\hahampis\Github\myproject\pom.xml" -Dtest=UsersRunner "-Dkarate.options=c:\Users\ChachampisN\Github\myproject\src\test\java\examples\users\users.feature"
```

However, if I copy this command and add `classpath:` after the `=` sign in `karate.options`:
```
cmd.exe /d /c mvn clean test -f "c:\Users\ChachampisN\Github\myproject\pom.xml" -Dtest=UsersRunner "-Dkarate.options=classpath:c:\Users\ChachampisN\Github\myproject\src\test\java\examples\users\users.feature"
```
then run it, it works.
Any ideas why adding `classpath:` makes a difference? Also in [Karate documentation](https://github.com/intuit/karate#command-line)  some examples are using this while some others don't...

karate version 0.9.6
karate runner 0.9.9
java 11
Windows 10

**Comment by kirksl** [comment]
@hahampis sorry for the delay.  i've been told that quickstart project might be out of date.  i also get the same error even when specifying `classpath`.  can you try the sample project here https://github.com/kirksl/karate-maven-gradle

i suspect this is a project configuration issue as the project i'm pointing you to works.

**Comment by hahampis** [comment]
@kirksl indeed, this one works.. thank you!

**Comment by hahampis** [comment]
@kirksl FYI I did some more digging on this since I couldn't let it go 😆 
I discovered that the error occurs when using the `@Karate.test` annotation from `com.intuit.karate.junit5.Karate` class. That's what's used in the quickstart project by default. On the other hand, in karate-maven-gradle project the `@Test` annotation from `org.junit.jupiter.api.Test` is used and the karate-runner extension works fine with such a test method. I have confirmed it also in my own project.
FWIW :)
